packages/daemon/src/graphql/objects.js

```diff
@@ -401,19 +401,22 @@ export const ownUserIdentity = new GraphQLObjectType({
       },
     },
     wallets: {
-      type: new GraphQLList(userWallet),
+      type: new GraphQLNonNull(ethWallets),
       resolve: (self, args, ctx) => {
-        const wallets = ctx.queries.getUserEthWallets(self.localID)
-        return Object.keys(wallets).map(id => {
-          return {
-            localID: id,
-            accounts: wallets[id].getAccounts(),
-          }
-        })
+        return ctx.queries.getUserEthWallets(self.localID)
       },
     },
     profile: {
-      type: new GraphQLNonNull(namedProfile),
+      type: new GraphQLObjectType({
```

Comment:
Any reason not to use `namedProfile` here instead?

Reply:
Must have been a merge error, will change it back